### PR TITLE
Dpdk backend: Initialize drop metadata to 1 as per PSA specification

### PR DESCRIPTION
**Pull request by usha1830**

### CODE REVIEW

**Review by github-sajan — APPROVED** [review]
looks good

File: backends/dpdk/dpdkProgram.cpp [hunk 1]
```diff
@@ -71,7 +71,7 @@ IR::IndexedVector<IR::DpdkAsmStatement> ConvertToDpdkProgram::create_psa_preambl
         new IR::Member(new IR::PathExpression("m"), "psa_ingress_input_metadata_ingress_port")));
     instr.push_back(new IR::DpdkMovStatement(
         new IR::Member(new IR::PathExpression("m"), "psa_ingress_output_metadata_drop"),
-        new IR::Constant(0)));
+        new IR::Constant(1)));
```
Comment (psivanup) [inline backends/dpdk/dpdkProgram.cpp:74]: Hope the constant type expected is "int". If not use explicit type (e.g. bit<8>) while creating constant.

**Review by psivanup — APPROVED** [review]
Looks fine.

File: backends/dpdk/dpdkProgram.cpp (hunk 1 above)
Comment (usha1830) [reply backends/dpdk/dpdkProgram.cpp:74]: This is right before assembly generation, no typechecking after this point.

Comment (psivanup) [reply backends/dpdk/dpdkProgram.cpp:74]: Ok

**Comment by usha1830** [comment]
Closing as upstreamed

**Pull request closed without merging**